dbtmetabase/metabase.py

```diff
@@ -140,6 +141,16 @@ def get_collection_items(
                 params={"models": models},
             )
         )
+        if exclude_unverified:
+            verified_results = list(
+                self._api(
+                    method="get",
+                    path=f"/api/search/",
+                    params={"models": models, "verified": True},
+                )
+            )
+            verified_ids = [y['id'] for y in verified_results]
+            results = [x for x in results if x['id'] in verified_ids]
```

Comment:
Unnecessary and wasteful, verified cards have a field `moderated_status` set to `"verified"`.

What happens to dashboards? They can't be verified, so will this just exclude all dashboards?